### PR TITLE
zh,en: Update deploy-tidb-cluster-across-multiple-kubernetes.md

### PR DESCRIPTION
**Pull request by july2993**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Fix wrong command in the docs

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
Hi @july2993, would you please specify the impacted TiDB Operator releases in the PR description and involve a tech reviewer? Thanks!

**Comment by july2993** [comment]
> Hi @july2993, would you please specify the impacted TiDB Operator releases in the PR description and involve a tech reviewer? Thanks!

no impacted version, it's just the docs is wrong.

@DanielZhangQD PTAL

**Comment by DanielZhangQD** [comment]
@july2993 @qiancai It only impacts the master version.

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Run the following command:
 {{< copyable "shell-regular" >}}
 
 ```bash
-cat << EOF | kubectl apply -f -n ${cluster1_namespace} -
+cat << EOF | kubectl apply -n ${cluster1_namespace} -f -
```
Comment (DanielZhangQD) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:60]: @july2993 Please also help update the file in `zh` dir, thanks!

**Comment by qiancai** [comment]
> @july2993 @qiancai It only impacts the master version.

@july2993，as I do not have the permission to edit your PR description to indicate that only the master branch is impacted, would you please help edit the PR description to make the **master (the latest development version)** check box selected? Thanks a lot!

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md (hunk 1 above)
Comment (july2993) [reply en/deploy-tidb-cluster-across-multiple-kubernetes.md:60]: https://github.com/pingcap/docs-tidb-operator/pull/1167/commits/4f4e1e32c73f64d7664ad5c13c3cdc7bc6c8da0c

File: tmp.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+  tikv:
```
Comment (yikeke) [inline tmp.yaml:1]: Do we need this file in the docs repo?  @DanielZhangQD

Comment (DanielZhangQD) [reply tmp.yaml:1]: NO. @july2993 Please remove this file.

**Comment by qiancai** [comment]
/retitle zh,en: Update deploy-tidb-cluster-across-multiple-kubernetes.md

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: tmp.yaml (hunk 2 above)
Comment (july2993) [reply tmp.yaml]: remove in https://github.com/pingcap/docs-tidb-operator/pull/1167/commits/bcb5d11f75b0b16b7c491a1f81c3b28a446e9fa4
sorry, it's by mistake

**Review by qiancai — APPROVED** [review]

**Review by yikeke — APPROVED** [review]

**Pull request merged**